### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by odebretseni**

### CODE REVIEW

File: tasks/install-java.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+- name: Download jdk.dmg
+  get_url:
+    url: https://github.com/frekele/oracle-java/releases/download/8u212-b10/jdk-8u212-macosx-x64.dmg
```
Comment (BrunIF) [inline tasks/install-java.yml]: JDK URL or version add to `default`

File: tasks/create-work-folder.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (BrunIF) [inline tasks/create-work-folder.yml]: needless

File: tasks/install-java.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    dest: /tmp
+
+- name: Mount the downloaded dmg
+  command: hdiutil attach -mountpoint /Volumes/jdk /tmp/jdk-8u212-macosx-x64.dmg
+
+- name: Install the pkg file from the dmg
+  become: true
+  become_method: sudo
+  command: installer -package /Volumes/jdk/JDK\ 8\ Update\ 212.pkg -target /
```
Comment (BrunIF) [inline tasks/install-java.yml]: version

**Review by BrunIF — CHANGES_REQUESTED** [review]

File: tasks/install-cordova.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+- name: Copy npm script
+  template: src=setup_npm_cordova.sh dest=/Users/{{ work_user }}/{{ work_dir }} mode=0777
+
+- name: Run npm script
+  command: sh /Users/{{ work_user }}/{{ work_dir }}/setup_npm_cordova.sh
+  tags: monit,skip_ansible_lint
```
Comment (BrunIF) [inline tasks/install-cordova.yml]: Why have you declared these tags?

File: tasks/install-java.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+- name: Download jdk.dmg
+  get_url:
+    url: https://github.com/frekele/oracle-java/releases/download/8u212-b10/jdk-8u212-macosx-x64.dmg
+    dest: /tmp
+
+- name: Mount the downloaded dmg
+  command: hdiutil attach -mountpoint /Volumes/jdk /tmp/jdk-8u212-macosx-x64.dmg
```
Comment (BrunIF) [inline tasks/install-java.yml]: version

File: templates/setup_npm_cordova.sh [hunk 6]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+PATH="$PATH:/usr/local/bin" npm install -g cordova@8
+
+echo -ne 'y\n' | PATH="$PATH:/usr/local/bin" sdkmanager "build-tools;27.0.3"
```
Comment (BrunIF) [inline templates/setup_npm_cordova.sh]: version?

**Review by BrunIF — CHANGES_REQUESTED** [review]

File: tasks/install-cordova.yml (hunk 4 above)
Comment (odebretseni) [reply tasks/install-cordova.yml]: when running lint on a playbook, it will not swear at the "command" module used

**Review by BrunIF — APPROVED** [review]

**Pull request merged**